packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -288,17 +288,17 @@ class _DraggableSheetExtent {
 class _DraggableScrollableSheetState extends State<DraggableScrollableSheet> {
   late _DraggableScrollableSheetScrollController _scrollController;
   late _DraggableSheetExtent _extent;
+  // The child only gets rebuilt when dependencies change. Otherwise, excessive
```

Comment:
nit: comment is out of date, also rebuilds on didUpdateWidget.

Reply:
Updated